demo-app/components/Ad.js

```diff
@@ -1,15 +1,10 @@
-import React from "react";
+import React from 'react';
```

Comment:
IMO we should avoid linting the `demo-app` itself as that's a separate project we're just bundling in. It seems to me that's unnecessary to lint it given it's only used for testing and it's maintained separately. That would also make linting quicker.

Reply:
Also updated .eslintrc too